packages/flutter_test/lib/src/binding.dart

```diff
@@ -705,8 +705,10 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
       // alone so that we don't cause more spurious errors.
       runApp(Container(key: UniqueKey(), child: _postTestMessage)); // Unmount any remaining widgets.
       await pump();
+      _pendingExceptionDetails = null;
       invariantTester();
-      _verifyAutoUpdateGoldensUnset(autoUpdateGoldensBeforeTest && !isBrowser);
+      _verifyAutoUpdateGoldensUnset(
+        autoUpdateGoldensBeforeTest && !isBrowser);
```

Comment:
looks like an accidental typo here

Reply:
what should it be?

Reply:
i assumed the newline was accidental. if it wasn't, then:
trivial nit: this is more readable when it's on one line than with the line break in the middle. :-)